auth0/src/test/java/com/auth0/android/provider/BrowserPickerTest.java

```diff
@@ -327,7 +328,7 @@ static void setupBrowserContext(@NonNull Context context, @NonNull List<String>
         PackageManager pm = mock(PackageManager.class);
         when(context.getPackageManager()).thenReturn(pm);
         ResolveInfo defaultPackage = resolveInfoForPackageName(defaultBrowserPackage);
-        when(pm.resolveActivity(any(Intent.class), anyInt())).thenReturn(defaultPackage);
+        when(pm.resolveActivity(any(Intent.class), eq(PackageManager.MATCH_DEFAULT_ONLY))).thenReturn(defaultPackage);
```

Comment:
better matchers, only by the values that we know we use. Will help catch changes like changing the prod code flags by mistake in the future 👍